### DISCUSSION TITLE
v0.23.0 - Attached field for Autogenerate feature

### DISCUSSION THREAD

**Discussion by hungcrush** [pocketbase/pocketbase#5627] [Q&A] [answered]
Hi,

Can we make Autogenerate follow another field? I'll call it the **Attached Field.**

It's similar to the slug field in WordPress.

**Example**:

**Input**: Xin chào các bạn
**Autogenerated value:** xin-chao-cac-ban
Thanks for your review.

**Comment by ganigeorgiev** [comment, marked as answer]
Not from the UI, but you can do this programmatically with a hook.

The "Autogenerate" field option operates only with regex patterns and was added mostly as a workaround for the existing system fields like `id` and `tokenKey` which with v0.23.0 are part of the collections fields list.

**Reply by hungcrush** [reply]
Thank you so much. I'm also thinking about hooks.
